packages/wallet-sdk/src/core/communicator/Communicator.ts

```diff
@@ -16,17 +17,19 @@ import { closePopup, openPopup } from ':util/web';
 export class Communicator {
   static communicators = new Map<string, Communicator>();
 
+  private readonly metadata: AppMetadata | undefined;
```

Comment:
let's make metadata not optional?

Reply:
Sounds goods. Let me update

Reply:
added here [6e64ca5](https://github.com/coinbase/coinbase-wallet-sdk/pull/1385/commits/6e64ca5a0f21bb3ccccdf5c01e7be8c8ce147101)